src/config/cluster-config.c

```diff
@@ -29,7 +31,7 @@ int Read_Cluster(XML_NODE node, void *d1, __attribute__((unused)) void *d2) {
     static const char *port = "port";
     static const char *bind_addr = "bind_addr";
     static const char *C_VALID = "!\"#$%&'-.0123456789:<=>?ABCDEFGHIJKLMNOPQRESTUVWXYZ[\\]^_abcdefghijklmnopqrstuvwxyz{|}~";
-
+    static const char *N_BLACKLIST = "!\"#$%&'-.,0123456789:<=>?ABCDEFGHIJKLMNOPQRESTUVWXYZ[\\]^_abcdefghijklmnopqrstuvwxyz{|}~";
```

Comment:
This file is actually used to limit which characters are used in options read by other services (for example, `node_name` being inserted into `global.db`). 

Since no service other than `wazuh-clusterd` uses the Agents reconnection info, I think there is no need to create a new list of characters. In fact, the validation should be done (if it is necessary) in the `wazuh-clusterd` process, since it can be started manually and the validation of this file would not be applied.

Reply:
We have created a list of characters since concatenating the string `C_VALID` with the `,` character was more expensive, also, we do this so that analysisd raises an exception in case it finds an error in the configuration:
```
2022/07/05 15:07:53 wazuh-db[13264] debug_op.c:70 at _log(): DEBUG: Logging module auto-initialized
2022/07/05 15:07:53 wazuh-db[13264] main.c:111 at main(): DEBUG: Wazuh home directory: /var/ossec
2022/07/05 15:07:53 wazuh-analysisd: ERROR: Detected a not allowed character in node name: "�master-node". Characters allowed: "!"#$%&'-.0123456789:<=>?ABCDEFGHIJKLMNOPQRESTUVWXYZ[\]^_abcdefghijklmnopqrstuvwxyz{|}~".
2022/07/05 15:07:53 wazuh-analysisd: ERROR: (1202): Configuration error at 'etc/ossec.conf'.
2022/07/05 15:07:53 wazuh-analysisd: CRITICAL: (1202): Configuration error at 'etc/ossec.conf'.
wazuh-analysisd: Configuration error. Exiting
```